packages/react-checkbox/src/components/Checkbox/Checkmark.tsx

```diff
@@ -0,0 +1,37 @@
+import * as React from 'react';
+
+export type CheckmarkProps = {
+  /**
+   * The width and height of the checkmark.
+   *
+   * @defaultvalue 8
+   */
+  size?: number;
+
+  /**
+   * Whether the checkmark is in a mixed/indeterminate state.
+   *
+   * @defaultvalue false
+   */
+  mixed?: boolean;
+
+  /**
+   * Whether the checkmark is in a circular checkbox. Only affects the mixed state.
+   *
+   * @defaultvalue false
+   */
+  circular?: boolean;
+};
+
+/**
+ * The default checkmark icon for Checkbox
+ */
+export const Checkmark = ({ size = 8, mixed, circular }: CheckmarkProps) => (
+  <svg width={size} height={size} viewBox="0 0 512 512" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Should this be one of the unsized icons that Tomi was talking about?

Reply:
The unsized icons don't look good in checkbox, and there's no good icon to use for the mixed state. After talking to Tomi, we're going to go with the fixed-size icons for now, and request a proper mixed icon (using "SquareFilled" for now, which has rounded corners that don't look good here).